docs/build/circuit-construction.ipynb

```diff
@@ -0,0 +1,486 @@
+      "text/plain": [
+       "<Figure size 538.33x367.889 with 1 Axes>"
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "qc_a.decompose().draw(\"mpl\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "649fc3fd-caf1-45f1-ad8e-3b5d26ca859b",
+   "metadata": {},
+   "source": [
+    "## Parameterized circuits\n",
+    "\n",
+    "Many near-term quantum algorithms repeatedly execute a circuit, then tweak the circuit based on the result. Using this process, these algorithms find circuits that produce specific outputs. The following cell shows a crude example of this: a loop that increases a gate angle until the circuit outputs `1` over 99% of the time."
```

Comment:
this could be a good place to call out the variational algos course in learning

Reply:
Good call, done!